8vim/src/main/res/drawable-anydpi/round_thumb.xml

```diff
@@ -2,8 +2,8 @@
 <shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
 
     <solid android:color="@android:color/white"/>
-    <corners android:radius="24dp"/>
-    <size android:height="28dp" android:width="28dp"/>
+    <corners android:radius="22dp"/>
+    <size android:height="25dp" android:width="25dp"/>
```

Comment:
Why do we still have hard coded numbers in the xml files and more importantly why do they change every commit.?